### DISCUSSION TITLE
一键长草-公开招募设置里添加优先刷新某一个公招位功能

### DISCUSSION THREAD

**Discussion by SlangSama** [MaaAssistantArknights/MaaAssistantArknights#5180] [Ideas]
4个公招位刷出高级资深干员的概率有差别，能否根据玩家的习惯添加刷新指定公招位的功能？

**Comment by ABA2396** [comment]
高资词条的出率只有0.2%，不知道是总结了多少次公招得出的不同公招概率有差别结论（）

**Reply by SlangSama** [reply]
理论跟实际完全一样吗？2700+次公招。我就是提个建议，没必要被阴阳怪气吧

**Reply by ABA2396** [reply]
我的意思不过是统计样本量在0.2%的概率下显得实在太少了。就算是每个槽位只统计10次出6星的情况，总的公招次数也得20000次左右，按照常见的规则每组30个观察值更得60000往上，就你说的2700次按0.2%算的话甚至不够给每个槽位分两个，又怎么得出4个公招位刷出高级资深干员的概率有差别的结论。

我估计群里是不会有人去做这个需求的，不过毕竟这是开源项目，你强烈需求的话自己写了提交pr也不会有人拦着你。
